project.clj

```diff
@@ -3,7 +3,7 @@
       [path]
       (str "resources/public/js/compiled/" path))
 
-(defproject onaio/chimera "0.0.9"
+(defproject onaio/chimera "0.0.11-SNAPSHOT"
```

Comment:
shouldn't this be `0.0.10-SNAPSHOT`, no?

Reply:
there is a `0.0.10` that is currently on clojars

Reply:
cool